packages/eslint-plugin/configs/react.js

```diff
@@ -35,5 +35,6 @@ module.exports = {
 		'react/prop-types': 'off',
 		'react/react-in-jsx-scope': 'off',
 		'react-hooks/rules-of-hooks': 'error',
+		'react-hooks/exhaustive-deps': 'warn',
```

Comment:
Nit: we could keep the list alphabetized. Sometimes that helps when reading the code.

Reply:
I'll fix this now.

Reply:
✅